www-client/seamonkey/seamonkey-2.53.11.ebuild

```diff
@@ -0,0 +1,584 @@
+		CHECKREQS_DISK_BUILD="16G"
+	else
+		CHECKREQS_DISK_BUILD="12G"
+	fi
+	check-reqs_pkg_setup
+}
+
+spkg_setup() {
+	# Ensure we have enough disk space to compile
+	if use debug || use lto || use test ; then
+		CHECKREQS_DISK_BUILD="16G"
+	else
+		CHECKREQS_DISK_BUILD="12G"
+	fi
+	check-reqs_pkg_setup
+}
+
+src_unpack() {
+	local l10n_sources="${P}.source-l10n.tar.xz"
+	unpack ${A/ ${l10n_sources}}
+
+	mkdir "${S}/${P}-l10n" || die
+	cd "${S}/${P}-l10n" || die
+	unpack ${l10n_sources}
+}
+
+src_prepare() {
+	SM_PATCHDIR="${WORKDIR}/gentoo-${PN}-patches-${PV}/${PN}"
+	# Apply our patches
+	eapply "${SM_PATCHDIR}/1000_fix-preferences-gentoo.patch"
+	eapply "${SM_PATCHDIR}/1001_gentoo_prefs.patch"
+	eapply "${SM_PATCHDIR}/1002_drop_build_id.patch"
+	eapply "${SM_PATCHDIR}/1003_gentoo_specific_pgo.patch"
+	eapply "${SM_PATCHDIR}/1004_fix_pie_detection.patch"
+	eapply "${SM_PATCHDIR}/1005_fix_fortify_sources.patch"
+	eapply "${SM_PATCHDIR}/1007_re-add_system_sqlite.patch"
+	eapply "${SM_PATCHDIR}/1008_seamonkey-2.53.10.2-remove_gtk2.patch"
+	eapply "${SM_PATCHDIR}/2000_system_harfbuzz.patch"
+	eapply "${SM_PATCHDIR}/2001_system_graphite2.patch"
+	eapply "${SM_PATCHDIR}/2002_bmo-1559213-Support-system-av1.patch"
+	eapply "${SM_PATCHDIR}/2003_nICER_implicit_decls.patch"
+	eapply "${SM_PATCHDIR}/2004_fix_lto_builds.patch"
+	eapply "${SM_PATCHDIR}/2006_musl_sys_auxv.patch"
+	eapply "${SM_PATCHDIR}/2009_upstream-bug_1461221.patch"
+	eapply "${SM_PATCHDIR}/2010_blessings-TERM.patch"
+	eapply "${SM_PATCHDIR}/2011_missing-errno_h-in-SandboxOpenedFiles_cpp.patch"
+	eapply "${SM_PATCHDIR}/2012_make-MOZ_SIGNAL_TRAMPOLINE-Android-only_bug1434526.patch"
+	eapply "${SM_PATCHDIR}/2014_dont_use_gconf_service_mozilla_bug1526243.patch"
+	eapply "${SM_PATCHDIR}/2015_dont_use_gconf_for_proxy_configuration_bug1540145.patch"
+	eapply "${SM_PATCHDIR}/2018_audio_device_backends.patch"
+	eapply "${SM_PATCHDIR}/6001_add_missing_header_for_basename.patch"
+	eapply "${SM_PATCHDIR}/6002_add_alternate_name_for_private_siginfo_struct_member.patch"
+	eapply "${SM_PATCHDIR}/6003_fix_syscall_wrappers_on_musl.patch"
+	eapply "${SM_PATCHDIR}/6004_musl_drop_alloc_hooks.patch"
+	eapply "${SM_PATCHDIR}/6005_musl_memory_report.patch"
+	eapply "${SM_PATCHDIR}/6006_musl_pthread_setname.patch"
+	eapply "${SM_PATCHDIR}/6007_musl_fix_tools.patch"
+	eapply "${SM_PATCHDIR}/6008_musl_fix_toolkit.patch"
+	eapply "${SM_PATCHDIR}/6018_audio_backend_update_generated_files.patch"
+
+	# Shell scripts sometimes contain DOS line endings; bug 391889
+	grep -rlZ --include="*.sh" $'\r$' . |
+	while read -r -d $'\0' file ; do
+		einfo edos2unix "${file}"
+		edos2unix "${file}"
+	done
+
+	use system-libvpx \
+		&& eapply -p2 "${SM_PATCHDIR}/1009_seamonkey-2.53.3-system_libvpx-1.8.patch"
+
+	# Allow user to apply any additional patches without modifing ebuild
+	eapply_user
+
+	# Don't error for format with gcc-9
+	grep -rl -- '-Werror=format' | xargs sed -i 's/error=format/no-&/' || die
```

Comment:
I'd prefer the `find -exec sed` syntax here but guess this piping is fine too.

Reply:
This is from the previous maintainer, I only added the "|| die" . I agree that a "find/sed"-way of doing this is better readable. I'm not sure if this line is still really needed as all build targets have moved on to gcc-11 by now and I don't encounter errors when building the development version of seamonkey manually (outside of portage). I'll have a closer look at this with the upcoming release.